src/machi.proto

```diff
@@ -48,9 +48,10 @@ enum Mpb_GeneralStatusCode {
      PARTITION = 4;
      NOT_WRITTEN = 5;
      WRITTEN = 6;
-     NO_SUCH_FILE = 7;
-     PARTIAL_READ = 8;
-     BAD_EPOCH = 9;
+     TRIMMED = 7; // The whole file was trimmed
```

Comment:
"The whole file was trimmed" ... hrm, I need to keep reading, but ... isn't `{error, trimmed}` also returned when only a byte range is trimmed?